base/util.jl

```diff
@@ -418,11 +433,18 @@ if is_windows()
 
         # Step 4: Free the encrypted buffer
         # ccall(:SecureZeroMemory, Ptr{Void}, (Ptr{Void}, Csize_t), outbuf_data[], outbuf_size[]) - not an actual function
+        unsafe_securezero!(outbuf_data[], outbuf_size[])
         ccall((:CoTaskMemFree, "ole32.dll"), Void, (Ptr{Void},), outbuf_data[])
 
-        # Done
-        Nullable((String(transcode(UInt8, usernamebuf[1:usernamelen[]-1])),
-            String(transcode(UInt8, passbuf[1:passlen[]-1]))))
+        # Done.  Fixme: for non-ascii passwords, transcode may leave
+        # an extra copy of the password in memory (due to resizing its buffer).
+        passbuf_ = passbuf[1:passlen[]-1]
```

Comment:
I'm wary of `resize!` because it doesn't make any guarantees about when it makes a copy of the buffer.